dynamo/configuration.py

```diff
@@ -432,25 +443,25 @@ def update_data_store_mode(mode: str) -> None:
 with warnings.catch_warnings():
     warnings.simplefilter("ignore")
     if "zebrafish" not in matplotlib.colormaps():
-        plt.register_cmap("zebrafish", zebrafish_cmap)
+        matplotlib.colormaps.register(name="zebrafish", cmap=zebrafish_cmap)
```

Comment:
can we use mpl here and the same for the rest?

Reply:
done